about.html

```diff
@@ -11,63 +11,110 @@
 
 </head>
 
+<div class="navContainer">
+  <img class="lambdaImage">
+  <img src="./img/lambda-black.png">
+  </img>
+  
+  
```

Comment:
If you want you can just use about.html and index.html for the links.  Either way is fine.  Later on when you get to React you will need to use ./ anyways to reference files in the same folder.